src/util/regex.h

```diff
@@ -1,6 +1,10 @@
 #pragma once
 
+#if QT_VERSION <= QT_VERSION_CHECK(5, 15, 0)
 #include <QRegExp>
```

Comment:
```
// QRegularExpression::escape() has been introduced in Qt 5.15.0
#if QT_VERSION >= QT_VERSION_CHECK(5, 15, 0)`
#include <QRegularExpression>
...
```

Reply:
done

Reply:
Please also switch the conditions as already mentioned for consistency. The legacy code should be secondary.

Reply:
done